### PR TITLE
Adjust the drop-position of the air strike in D2k

### PR DESCRIPTION
**Pull request by abc013**
Before change

d2kbug
---
After change

d2kbugfix

### CODE REVIEW

**Review by GraionDilach — APPROVED** [review]

File: mods/d2k/weapons/other.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ OrniBomb:
 	ReloadDelay: 25
 	Burst: 5
 	BurstDelay: 3
-	Range: 3c0
+	Range: 1c500
```
Comment (Phrohdoh) [inline mods/d2k/weapons/other.yaml]: `1c512` is half of `3c0`.

**Comment by abc013** [comment]
Updated.

**Comment by reaperrr** [comment]
This can be improved further, for example ornithopters currently fly too low and the bombs fall too fast, lack shadow and play the wrong sound on impact.

I've filed a PR to your branch that fixes these and slightly increases the distance between them.

In the future we should further increase the delay between them, but we need linear formation support and multi-flybys first, othwise they deal too little damage to the target position.

**Comment by GraionDilach** [comment]
I have started working on multi-flybys but my method involves dropping AttackBomber out and I haven't looked at the Lua requirements to see what upstream requires for an 1:1 switch to my setup (I also have some bugs with guarding anyways).

**Comment by abc013** [comment]
@reaperrr looks good, merged it.

One question: in the original game, do the explosions of the bombs create craters?

**Comment by abc013** [comment]
d2kbugfix gif

**Comment by reaperrr** [comment]
> One question: in the original game, do the explosions of the bombs create craters?

Yes, they did, I just forgot to check for that.

**Comment by reaperrr** [comment]
As it turns out, the craters are not appearing because the OrniBomb weapon (other.yaml) defines a bogus 'Crater' smudge type. The correct setting would be `SmudgeType: SandCrater, RockCrater`

Additionally, I'd suggest to add `SquadOffset: -1536, 1024, 0` to the AirstrikePower, to make the attacked area exactly 3x3 cells (in the above gif, the outer ornis would drop their bombs in the middle of the turret cells instead of on the outer border).

👍 after that.

**Comment by abc013** [comment]
d2kbugfix3

Are you sure you dont mean 4x3 instead of 3x3?

**Comment by reaperrr** [comment]
> Are you sure you dont mean 4x3 instead of 3x3?

You're right, of course, I was only thinking about the "width".

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
@GraionDilach Just to be sure, do you consider your +1 still valid after the changes?

**Review by GraionDilach — APPROVED** [review]
I didn't. Now I do.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d703ee5d46968382725567db546c4449ecdeab3a...190fae97a20f105f49e27d2ea0af3e5c3731d954)

**Pull request merged**